src/funnels/StableSwapper.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-FileCopyrightText: © 2020 Lev Livnev <lev@liv.nev.org.uk>
```

Comment:
We need to discuss the directory structure here.
Probably the automation samples should be in a different directory thab the base level primitives.

Reply:
resolving as I see this was just done